src/components/Card.js

```diff
@@ -27,17 +31,51 @@ export default class Card {
     this._cardImageElement.alt = this.name;
     this._likeButton = this._element.querySelector(".card__like-button");
     this._deleteButton = this._element.querySelector(".card__delete-button");
-
+    this._checkLikeStatus();
     this._setEventListeners();
     return this._element;
   }
 
   _handleLikeButton() {
     this._likeButton.classList.toggle("card__like-button_active");
+    if (this.isLiked) {
+      api.removeLike(this._id).catch((err) => console.log(err));
+      this.isLiked = false;
```

Comment:
You need to **toggle likes** only in a `then` block after a successful response from the server because if there is an error, the user will not know about that and will see the toggled heart color.